addons/website_sale/models/product_template.py

```diff
@@ -157,18 +157,20 @@ def _sort_key_variant(variant):
 
         return self._get_possible_variants(parent_combination).sorted(_sort_key_variant)
 
-    def _get_sales_prices(self, pricelist):
-        pricelist.ensure_one()
+    def _get_sales_prices(self, pricelist=None):
```

Comment:
why the None ?
it's called only in one place, and a recordset should always be given (empty or not) :thinking: